data-prepper-core/src/main/java/org/opensearch/dataprepper/core/sourcecoordination/enhanced/EnhancedLeaseBasedSourceCoordinator.java

```diff
@@ -112,6 +112,12 @@ public <T> boolean createPartition(EnhancedSourcePartition<T> partition) {
                 // For now, global items with no partitionType will be considered ReadOnly, but this should be directly in EnhancedSourcePartition in the future
                 partition.getPartitionType() == null);
 
+        if (created) {
+            LOG.info("Created partition item with source identifier {} and partition key {}",
+                    this.sourceIdentifier + "|" + partitionType, partition.getPartitionKey());
```

Comment:
Please create a variable for this so that it is tightly coupled with the actual value used on line 107:

```
this.sourceIdentifier + "|" + partitionType
```